### ISSUE TITLE
Startup Scripts no longer work since the permission overhaul introduced in NetBox 2.9.0

### ISSUE THREAD

**Issue by ryanmerolle** [netbox-community/netbox-docker#449]
## Desired Behavior

1. Define a permission
2. Associate permission to a user or group object.

## Contrast to Current Behavior

Current Startup Script for groups references a defunct permissions model where permissions within the user or group objects.

## Changes Required

Updates required to the following files:

- startup_scripts/000_permissions.py (new file)
- startup_scripts/000_users.py
- startup_scripts/010_groups.py
- initializers/permissions.yml
- initializers/groups.yml
- initializers/users.yml

## Discussion: Benefits and Drawbacks

Would align with NetBox versions 2.9.x and 2.10.x models

**Comment by ryanmerolle** [comment]
I'll look to submit a PR in the next day or two.

**Comment by ryanmerolle** [comment]
I am having trouble dealing with the object_permissions.py

The data model for ObjectPermissions in the users model (per the API) expects:
```
{
  "name": "string",
  "description": "string",
  "enabled": true,
  "object_types": [
    "string"
  ],
  "groups": [
    0
  ],
  "users": [
    0
  ],
  "actions": [
    "string"
  ],
  "constraints": "string"
}
```

I convert the list of actions or list of object types and it kicks back that it should be a string and not a list.  I convert the param to a string and it does not like it either.  I will get my draft PR up shortly.  Its just a little challenging to reverse engineer given my expertise and inability to model against a similar startup script that has a list param requirement.

**Comment by cimnine** [comment]
I would recommend to look at the respective models in NetBox's Django code. You're not dealing with the REST API in the startup_scripts, but with the actual model objects of NetBox. While the REST API gives you hints, only the actual models in NetBox's Django code are authoritative.

**Comment by ryanmerolle** [comment]
Yep, I generally interact with the API and up until this point, my updates for startup_scripts have been based of that and reverse engineering existing scripts.

After I wrote this I jumped into the netbox shell.

My main problem is that when pass a list of users( `["user1", "user2"]` ) to the users attribute of the ObjectPermission model, it complains that a list of strings was given instead of ids.

I'll have to play around with efficiently taking said list and get the associated id to feed it.
